packages/miniflare/test/index.spec.ts

```diff
@@ -303,9 +307,13 @@ test("Miniflare: custom service using Content-Encoding header", async (t) => {
 
 	const test = async (encoding: string) => {
 		const res = await mf.dispatchFetch("http://localhost", {
-			headers: { "X-Test-Encoding": encoding },
+			headers: {
+				"Accept-Encoding": encoding,
+				"X-Test-Encoding": encoding,
+			},
 		});
 		t.is(await res.text(), testBody, encoding);
+		t.is(res.headers.get('Content-Encoding'), encoding, encoding);
```

Comment:
I noticed this was test wasn't actually testing anything – the encoding was always "gzip". I added this assertion and it was failing – I had to add the "Accept-Encoding" header above so we are now actually testing each encoding